rrmngmnt/nmcli.py

```diff
@@ -237,6 +221,7 @@ def add_ethernet_connection(
         ipv6_method=None,
         ipv6_addr=None,
         ipv6_gw=None,
+        **kwargs
```

Comment:
kwargs hide a lot of things, any reason to use it and not specify the args that you need here?